lib/rets/client.rb

```diff
@@ -9,11 +9,13 @@ class Client
 
     def initialize(options)
       @options = options
-      @capabilities = nil
       clean_setup
     end
 
     def clean_setup
+      if options.fetch(:login_after_error, false)
```

Comment:
@hfaulds @dougcole I think I would argue for keeping this simpler with just `@capabilities = nil` and not having the `:login_after_error` option as it was before #136 . That is, of course, unless you guys know of a specific use case where this option is necessary. Otherwise, I would guess the option won't be used much and will be unnecessary code. Better to add the option when necessary and a specific need comes about.